danskernesdigitalebibliotek/dpl-cms
# DDFFORM 475

#### Link to issue
[DDFFORM-475](https://reload.atlassian.net/browse/DDFFORM-475)

#### Description

This PR does several things. At large this is: 

- Replace old view.event_admin with view.event_series_admin which shows the event series on /admin/content/events. 
- Create a tab on the eventseries edit page, showing a list of all eventinstances in that seris. 
- Enforces that tab is only showed conditionally, see commits for info. 
- Create eventsubcriber for redirecting both editors and users on eventinstances/eventseries conditionallity. See commits for info. 
- Disable the ability of cloning eventseries / eventinstances. 
- Disables the possiblity of viewing revisions for eventseries / eventinstances. 


#### Manual test

Test that you; 
- Cannot access the frontend of an event series with only 1 instance. (you are redirected)
- Cannot access the editing page of an event instance, which is the only instance of the series. (You are redirected) 
- Can see a view of eventseries, which relevant information correctly formatted and with necessary links. 
- Can see a tab for "Edit instances" for series that have more than one eventinstance. 
- Can see a view of eventinstances under the "edit instances" tab, which relevant information correctly formatted and with necessary links. 
- Cannot see "clone" or "revisions" as a non-superadmin user. 



#### Screenshot of the result

New series view: 
![image](https://github.com/danskernesdigitalebibliotek/dpl-cms/assets/13272656/2b7886b4-ac2a-43fe-be13-6c0947190384)


New instance view: 
![image](https://github.com/danskernesdigitalebibliotek/dpl-cms/assets/13272656/8772903c-83d1-4730-8a6a-6840a2437e38)


#### Additional comments or questions

Please carefully consider if anything has been missed or overlooked in the replacement/removal of the old view for events. 

Also consider if there are any permissions or general view settings that have been missed. 

[DDFFORM-475]: https://reload.atlassian.net/browse/DDFFORM-475?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
